src/tests/ftest/util/ior_test_base.py

```diff
@@ -56,41 +56,36 @@ def create_pool(self):
         # Get the pool params and create a pool
         self.add_pool(connect=False)
 
-    def create_cont(self, chunk_size=None, properties=None):
-        """Create a TestContainer object to be used to create container.
+    def get_container(self, pool=None, namespace=None, create=True, **kwargs):
```

Comment:
(pylint-arguments-differ) Number of parameters was 6 in 'TestWithServers.get_container' and is now 5 in overridden 'IorTestBase.get_container' method